extensions/wikia/UserLogin/FacebookSignupController.class.php

```diff
@@ -97,19 +114,15 @@ public function modal() {
 		}
 
 		// get an email from Facebook API
-		$resp = $this->sendRequest( 'FacebookSignup', 'getFacebookData', [
-			'fbUserId' => $fbUserId,
-		] );
+		$email = \FacebookClient::getInstance()->getEmail( $fbUserId );
 
 		// BugId:24400
-		$data = $resp->getData();
-		if ( empty( $data ) ) {
+		if ( !$email ) {
```

Comment:
Do we want to skip rendering if there's no email? User's can sign up for facebook using a phone number. It seems like this would block those users from being able to sign up.